### ISSUE TITLE
create-env is broken with v68

### ISSUE THREAD

**Issue by FlorianNachtigall** [cloudfoundry-incubator/bosh-aws-cpi-release#74]
We are using `bosh-deployment` with latest versions. Since yesterday we are not able to execute `create-env` with AWS CPI anymore. We are using `access_key_id` and `secret_access_key`. Based on the release notes for the bumped AWS CPI v68, nothing should change for us.

The error message we get is:
```
Deploying:
  Creating instance 'bosh/0':
    Creating VM:
      Creating vm with stemcell cid 'ami-304dcf5f light':
        CPI 'create_vm' method responded with error: CmdError{"type":"Unknown","message":"You are not authorized to perform this operation.","ok_to_retry":false}

Exit code 1
```

Is this related to the new [optional session-token](https://github.com/cloudfoundry-incubator/bosh-aws-cpi-release/commit/a74f5aead781c3edaeb3edbb66eae092d4ae6569#diff-62672db1c125f6c3548a1263ea02b037R23)? We did not specify it as it says its optional.

We are not modifying `credentials_source ` so the default value `static` applies.
Does the documentation for `session-token` (Optional, used when aws.credentials_source is set to 'static') mean "it 'might' be used with aws.credentials_source=static" or does it rather mean "you 'have to' use it when aws.credentials_source=static"?

**Comment by FlorianNachtigall** [comment]
We just figured out adding the IAM policies from 5f881b8813dd2a54326c2217823ba7ffc6a893b6 solves our issue.
```
- ec2:CopyImage
- ec2:ModifyInstanceAttribute
- ec2:DescribeAvailabilityZones
```

Isn't this a breaking change which should be mentioned in the release notes?

**Comment by cdutra** [comment]
ModifyInstanceAttribute -> this was an error in v68 that required this permission even if `source_dest_check` was set to true in your manifest. It will not be necessary in the next release as we have fixed the logic around that.
CopyImage -> if uploading encrypted light stemcells
DescribeAvailabilityZones -> It always needed

**Comment by cdutra** [comment]
Fixed in 10231a5f53c9f29b8873030ee9c054a667f8bec3
